x-pack/plugins/dataset_quality/server/routes/data_streams/routes.ts

```diff
@@ -7,14 +7,21 @@
 
 import * as t from 'io-ts';
 import { keyBy, merge, values } from 'lodash';
-import { dataStreamTypesRt } from '../../../common/data_streams';
-import { DataStreamsStatResponse } from '../../types/data_stream';
+import {
+  DATA_STREAMS_MALFORMED_STATS_URL,
+  DATA_STREAMS_STATS_URL,
+} from '../../../common/constants';
+import { DataStreamStat } from '../../types/data_stream';
+import { dataStreamTypesRt, rangeRt } from '../../types/default_api_types';
+import { Integration } from '../../types/integration';
 import { createDatasetQualityServerRoute } from '../create_datasets_quality_server_route';
 import { getDataStreams } from './get_data_streams';
 import { getDataStreamsStats } from './get_data_streams_stats';
+import { getMalformedDocsPaginated } from './get_malformed_docs';
+import { MalformedDocs } from '../../../common/api_types';
 
 const statsRoute = createDatasetQualityServerRoute({
-  endpoint: 'GET /internal/dataset_quality/data_streams/stats',
+  endpoint: `GET ${DATA_STREAMS_STATS_URL}`,
```

Comment:
Do we really need this abstraction ? it makes navigating code harder. Generally we would copy the endpoint from the frontend and then search for it to find the server code like in APM

Reply:
I would favour hardcoding links over consts. The gain of consts is simply not worth the pain of not being able to navigate the code.

Reply:
I'll explicitly add the routes there then, also easier to find API tests in that way

Reply:
Addressed in [6df80a7](https://github.com/elastic/kibana/pull/172462/commits/6df80a76019d12292af6e40d37c9650da4b0363e).